lib/ui/screens/home_screen.dart

```diff
@@ -71,6 +71,7 @@ class _HomeScreenState extends State<HomeScreen> {
           ],
         ),
         body: PageView(
+          physics: NeverScrollableScrollPhysics(),
```

Comment:
this would make clicking on the bottom taps having no animation

Reply:
pls apply the fix like this:
https://api.flutter.dev/flutter/widgets/AndroidView/gestureRecognizers.html